README.md

```diff
@@ -15,155 +15,16 @@ run on Amazon Linux 2:
 - java11
 
-## Version ARNs
+## Version Numbers
 
-<details>
-<summary>Ghostscript v10.0.0
-</summary>
-
-| AWS Region     | Layer ARN                                                         |
-| -------------- |-------------------------------------------------------------------|
-| us-east-1      | `arn:aws:lambda:us-east-1:764866452798:layer:ghostscript:13`      |
-| us-east-2      | `arn:aws:lambda:us-east-2:764866452798:layer:ghostscript:13`      |
-| us-west-1      | `arn:aws:lambda:us-west-1:764866452798:layer:ghostscript:13`      |
-| us-west-2      | `arn:aws:lambda:us-west-2:764866452798:layer:ghostscript:13`      |
-| ca-central-1   | `arn:aws:lambda:ca-central-1:764866452798:layer:ghostscript:13`   |
-| sa-east-1      | `arn:aws:lambda:sa-east-1:764866452798:layer:ghostscript:13`      |
-| eu-north-1     | `arn:aws:lambda:eu-north-1:764866452798:layer:ghostscript:13`     |
-| eu-central-1   | `arn:aws:lambda:eu-central-1:764866452798:layer:ghostscript:13`   |
-| eu-west-1      | `arn:aws:lambda:eu-west-1:764866452798:layer:ghostscript:13`      |
-| eu-west-2      | `arn:aws:lambda:eu-west-2:764866452798:layer:ghostscript:13`      |
-| eu-west-3      | `arn:aws:lambda:eu-west-3:764866452798:layer:ghostscript:14`      |
-| ap-northeast-1 | `arn:aws:lambda:ap-northeast-1:764866452798:layer:ghostscript:13` |
-| ap-northeast-2 | `arn:aws:lambda:ap-northeast-2:764866452798:layer:ghostscript:13` |
-| ap-south-1     | `arn:aws:lambda:ap-south-1:764866452798:layer:ghostscript:13`     |
-| ap-southeast-1 | `arn:aws:lambda:ap-southeast-1:764866452798:layer:ghostscript:13` |
-| ap-southeast-2 | `arn:aws:lambda:ap-southeast-2:764866452798:layer:ghostscript:13` |
-</details>
-
-<details>
-<summary>Ghostscript v9.56.1
-</summary>
-
-| AWS Region     | Layer ARN                                                         |
-| -------------- |-------------------------------------------------------------------|
-| us-east-1      | `arn:aws:lambda:us-east-1:764866452798:layer:ghostscript:12`      |
-| us-east-2      | `arn:aws:lambda:us-east-2:764866452798:layer:ghostscript:12`      |
-| us-west-1      | `arn:aws:lambda:us-west-1:764866452798:layer:ghostscript:12`      |
-| us-west-2      | `arn:aws:lambda:us-west-2:764866452798:layer:ghostscript:12`      |
-| ca-central-1   | `arn:aws:lambda:ca-central-1:764866452798:layer:ghostscript:12`   |
-| sa-east-1      | `arn:aws:lambda:sa-east-1:764866452798:layer:ghostscript:12`      |
-| eu-north-1     | `arn:aws:lambda:eu-north-1:764866452798:layer:ghostscript:12`     |
-| eu-central-1   | `arn:aws:lambda:eu-central-1:764866452798:layer:ghostscript:12`   |
-| eu-west-1      | `arn:aws:lambda:eu-west-1:764866452798:layer:ghostscript:12`      |
-| eu-west-2      | `arn:aws:lambda:eu-west-2:764866452798:layer:ghostscript:12`      |
-| eu-west-3      | `arn:aws:lambda:eu-west-3:764866452798:layer:ghostscript:10`      |
-| ap-northeast-1 | `arn:aws:lambda:ap-northeast-1:764866452798:layer:ghostscript:12` |
```

Comment:
Unfortunately, not so easy here. As you see, depending on a region, layer ARN version might be different. This is because we added support for new regions at a different point of time. Maybe extend your table with columns like "All Regions ARN version" + "eu-west-3 ARN version"?

Reply:
On the second thought. Let's remove reference to older versions. New users should be interested in the latest version of the layer only.

Reply:
Edited the table to accommodate eu-west-3 discrepancy